nginx/etc/nginx/conf.d/tiler.conf

```diff
@@ -15,7 +15,7 @@ server {
 	include /etc/nginx/includes/tiler-security-headers.conf;
 
 	location ~ /tiles/(.*) {
-		return 301 $scheme://$http_host/$1$is_args$args;
+		rewrite /tiles/(.*) /$1 last;
```

Comment:
Instead of rewriting all requests, I think that we should try to rewrite only the paths and methods that are required for the preflight requests, and redirect everything else.

Do the CORS requests that are failing now have a more specific pattern?